pandas/tests/indexes/common.py

```diff
@@ -339,7 +339,7 @@ def test_memory_usage(self, indices):
 
         # RangeIndex, IntervalIndex
```

Comment:
can you update / change the comment to more reflect what is going on (i would remove the references to RI and II here)

Reply:
```
indices = PeriodIndex(['2000-01-03', '2000-01-04', '2000-01-05', '2000-01-06',
             '2000-01-07', '2000-01-10', '2000-01-11', '2000-01-12',
             '2000-01-13', '2000-01-14'],
            dtype='period[D]', freq='D')
result = indices.memory_usage()
indices.get_loc(indices[0])
result2 = indices.memory_usage()
```

"result" and "result2" same as 80